modules/settings.js

```diff
@@ -12,64 +11,137 @@ function Settings() {
     repeat: true
   };
 
+  // Are these the paths we want?
+  this.backupPaths = {
+    apps: 'backup/apps/',
+    music: 'backup/music/',
+    photos: 'backup/photos/',
+    videos: 'backup/videos/',
+    contacts: 'backup/videos/',
+    settings: 'backup/settings/'
+  };
+
+  // Load persistent settings from local storage, if they exist
+  this.load();
+}
+
+Settings.prototype.load = function() {
   // Load options if present
-  (function() {
-    var retrievedOptions = localStorage.getItem('ffosbrOptions');
+  var retrievedOptions = localStorage.getItem('ffosbrOptions');
 
-    if (retrievedOptions !== null) {
-      self.options = JSON.parse(retrievedOptions);
-    }
-  })();
+  if (retrievedOptions !== null) {
 
-  Settings.prototype.options = function(myDictionary) {
-    if (!myDictionary) {
-      return options;
-    }
+    retrievedOptions = JSON.parse(retrievedOptions);
```

Comment:
For now, I just have it throwing errors. However, I'm not sure this is the best approach. So let me know.